spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb

```diff
@@ -114,7 +114,7 @@ def assert_specific_container
           @container.last_started_at,
           @container.last_finished_at,
         ].each do |date_|
-          expect(date_.kind_of?(ActiveSupport::TimeWithZone) || date_.kind_of?(NilClass)).to be_truthy
+          expect(date_.kind_of?(ActiveSupport::TimeWithZone))
```

Comment:
Was this change intentional?  This expectation is no longer expecting anything?

```suggestion
          expect(date_.kind_of?(ActiveSupport::TimeWithZone)).to be
```

or

```suggestion
          expect(date_.kind_of?(ActiveSupport::TimeWithZone)).to be_truthy
```

Reply:
I'm actually just going to drop the whole loop since `expect time or nil` is what hid this bug in the first place